addons/mail/static/src/messaging/widget/form_renderer/form_renderer_tests.js

```diff
@@ -349,11 +334,9 @@ QUnit.test('chatter updating', async function (assert) {
         1,
         "there should be a message"
     );
-    assert.verifySteps(['message_fetch:2']);
```

Comment:
why remove? can be preped?

Reply:
removed because it doesn't make sense to assert 1st or 2nd message fetch, and it happens in prep because test uses a counter while improved test doesn't.